<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSSDlg</class>
 <widget class="QDialog" name="SSSDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shamir Secret Sharing Scheme</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="mSrcLenText">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Source Data</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mSrcTypeCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="mSrcText"/>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="mAddBtn">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Share Data</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mShareText"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="mShareTable">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Joined Data</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="mJoinedLenText">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mJoinedTypeCombo"/>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="mJoinedText">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="mNText">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>K</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="mJoinBtn">
       <property name="text">
        <string>Join</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="mKText">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="mSplitBtn">
       <property name="text">
        <string>Split</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mCloseBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="mClearBtn">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mSrcTypeCombo</tabstop>
  <tabstop>mSrcLenText</tabstop>
  <tabstop>mSrcText</tabstop>
  <tabstop>mShareText</tabstop>
  <tabstop>mAddBtn</tabstop>
  <tabstop>mShareTable</tabstop>
  <tabstop>mJoinedTypeCombo</tabstop>
  <tabstop>mJoinedLenText</tabstop>
  <tabstop>mJoinedText</tabstop>
  <tabstop>mNText</tabstop>
  <tabstop>mKText</tabstop>
  <tabstop>mSplitBtn</tabstop>
  <tabstop>mJoinBtn</tabstop>
  <tabstop>mClearBtn</tabstop>
  <tabstop>mCloseBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
